ui/src/styles/layout/element-plus-overload.scss

```diff
@@ -133,6 +133,9 @@
     }
 }
 
+.el-button + .el-dropdown, .el-dropdown + .el-button {
+    margin-left: 12px;
```

Comment:
Is 12 a magic number ?

Reply:
I went the Bootstrap way instead - no more weird constants.